ckanext/cloudstorage/logic/action/multipart.py

```diff
@@ -0,0 +1,269 @@
+import ckan.lib.helpers as h
```

Comment:
Missing encoding line, `# -*- encoding: utf-8 -*-`

Imports should always be ordered for consistency:

1. std modules (datetime, logging, etc...), newline
2. 3rd party (ckan, pylons, etc...), newline
3. current module (from ckanext.cloudstorage...)